remote-finetuning.md

```diff
@@ -0,0 +1,128 @@
+2. Make sure you have a [HuggingFace account](https://huggingface.co/) and [generate an access token](https://huggingface.co/docs/hub/security-tokens) if you are using private dataset on HuggingFace or your base model needs access control.
+3. Accept the LICENSE on HuggingFace if you are fine-tuning Mistral or Llama. 
+4. If you enable [Weights & Biases](https://wandb.ai/site), make sure you have a Weights & Biases account and [retrieve your API Key](https://wandb.ai/authorize)
+
+## Provision Azure Resources
+To get started, you need provision the Azure Resource for remote fine-tuning. This can be done by running the `AI Toolkit: Provision Azure Container Apps job for fine-tuning.` from command palette. During this process, you will be prompted to select your Azure Subscription and resource group.
+
+![Provision Fine-Tuning](Images/remote/provision-finetune.png)
+
+You can monitor the progress of the provision via the link that is displayed in the output channel.
+![Provision Progress](Images/remote/provision-progress.png)
+
+## Run
+To start the remote fine-tuning job, execute the `AI Toolkit: Run fine-tuning` command.
+![Run Fine-tuning](Images/remote/run-finetuning.png)
+
+Upon running this command, the extension will do the following operations:
+1. Synchronize your workspace with Azure Files.
+1. Start the ACA job using the commands specified in `./infra/fintuning.config.json`.
+1. Display the job streaming log if the job has started running. 
+    ![Run Log](Images/remote/run-log.png)
+    > **Note:** The job might be queued if there are insufficient resources available. If the log fails to display when the job starts, you can wait for a while and then execute the `AI Toolkit: Show the running fine-tuning job streaming logs` command to re-connect to the streaming log.
+    
+During this process, QLoRA will be used for fine-tuning, and will create LoRA adapters for the model to use during inference.
+The results of the fine-tuning will be stored in the Azure Files.
+
+## View and Query Logs on Azure
+
+After fine-tuning job was triggered, you can also view logs on Azure by clicking the "*Open Logs in Azure Portal*" button from the VSCode notification.
+
+Or, if you already opened the Azure Portal, you can find job history from the "*Execution history*" panel to the Azure Container Apps job.
+
+![Job Execution History](Images/remote/finetune-job-history.png)
+
+There are two types of logs, "*Console*" and "*System*".
+- Console logs, which are emitted by your app, including `stderr` and `stdout` messages. That's also what you may already see in previous section's streaming logs.
```

Comment:
Proposed: 
Console logs are messages from your app, including stderr and stdout messages. This is what you might have already seen in the streaming logs section.

Reply:
Updated